src/simplewallet/simplewallet.cpp

```diff
@@ -1040,8 +1040,6 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
       }
       crypto::secret_key viewkey = *reinterpret_cast<const crypto::secret_key*>(viewkey_data.data());
 
-      m_wallet_file=m_generate_from_view_key;
```

Comment:
Because the same thing appears at [#L1011](https://github.com/kenshi84/monero/blob/db8cfa220a99ba6d61f2a087b2db1c6585507c25/src/simplewallet/simplewallet.cpp#L1011). Isn't this line redundant?